plugins/zenoh-plugin-storage-manager/src/replica/digest.rs

```diff
@@ -418,11 +436,11 @@ impl Digest {
                 .subintervals
                 .entry(subinterval)
                 .and_modify(|e| {
-                    e.content.insert(ts.clone());
+                    e.content.insert(log_entry.clone());
                 })
                 .or_insert(SubInterval {
                     checksum: 0,
-                    content: [ts].iter().cloned().collect(),
+                    content: [log_entry].iter().cloned().collect(),
```

Comment:
Can be replaced by `content: [log_entry].into()`
(and the same with `Interval`s' content in following lines)